event/listener.php

```diff
@@ -279,6 +280,45 @@ public function validate_pwa_options($event)
 		}
 	}
 
+	/**
+	 * Add Web Push info to the phpBB FAQ
+	 *
+	 * @param \phpbb\event\data $event The event object
+	 * @return void
+	 */
+	public function wpn_faq($event)
+	{
+		if ($event['block_name'] === 'HELP_FAQ_BLOCK_ATTACHMENTS')
+		{
+			$this->language->add_lang('webpushnotifications_faq', 'phpbb/webpushnotifications');
+
+			$this->template->assign_block_vars('faq_block', [
+				'BLOCK_TITLE'	=> $this->language->lang('HELP_FAQ_WPN'),
+				'SWITCH_COLUMN'	=> false,
+			]);
+
+			$questions = [
+				'HELP_FAQ_WPN_WHAT_QUESTION'    => 'HELP_FAQ_WPN_WHAT_ANSWER',
+				'HELP_FAQ_WPN_HOW_QUESTION'     => 'HELP_FAQ_WPN_HOW_ANSWER',
+				'HELP_FAQ_WPN_IOS_QUESTION'     => 'HELP_FAQ_WPN_IOS_ANSWER',
+				'HELP_FAQ_WPN_SESSION_QUESTION' => 'HELP_FAQ_WPN_SESSION_ANSWER',
+				'HELP_FAQ_WPN_SUBBING_QUESTION' => 'HELP_FAQ_WPN_SUBBING_ANSWER',
+				'HELP_FAQ_WPN_GENERAL_QUESTION' => 'HELP_FAQ_WPN_GENERAL_ANSWER',
+			];
+
+			$faq_rows = [];
+			foreach ($questions as $question => $answer)
+			{
+				$faq_rows[] = [
+					'FAQ_QUESTION' => $this->language->lang($question),
+					'FAQ_ANSWER'   => $this->language->lang($answer),
+				];
+			}
+					
+			$this->template->assign_block_vars_array('faq_block.faq_row', $faq_rows);
+		}
+	}
```

Comment:
I feel like we're duplicating the code from the help manager here. Instead we could just inject the manager service and use it like f.e.
```php
$this->language->add_lang('webpushnotifications_faq', 'phpbb/webpushnotifications');
$questions = [
  'HELP_FAQ_WPN_WHAT_QUESTION'    => 'HELP_FAQ_WPN_WHAT_ANSWER',
  'HELP_FAQ_WPN_HOW_QUESTION'     => 'HELP_FAQ_WPN_HOW_ANSWER',
  'HELP_FAQ_WPN_IOS_QUESTION'     => 'HELP_FAQ_WPN_IOS_ANSWER',
  'HELP_FAQ_WPN_SESSION_QUESTION' => 'HELP_FAQ_WPN_SESSION_ANSWER',
  'HELP_FAQ_WPN_SUBBING_QUESTION' => 'HELP_FAQ_WPN_SUBBING_ANSWER',
  'HELP_FAQ_WPN_GENERAL_QUESTION' => 'HELP_FAQ_WPN_GENERAL_ANSWER',
];
$this->help_manager->add_block('HELP_FAQ_WPN', false, $questions);
```

Reply:
Yeah but I dont want to write the tests for that just to eliminate a few lines of code in here